Packages/MIES/MIES_WaveBuilder.ipf

```diff
@@ -454,7 +454,7 @@ End
 /// @sa AddDimLabelsToWP()
 static Function/WAVE WB_GetControlWithDeltaIdx()
 
-	Make/FREE/B indizes = {0, 2, 4, 6, 10, 12, 14, 16, 20, 22, 24, 26, 28, 30, 45, 99}
+	Make/FREE/B indizes = {0, 2, 4, 6, 8, 10, 12, 14, 16, 20, 22, 24, 26, 28, 30, 45, 99}
```

Comment:
@t-b At some point down the road, what do you think about making these (static) constants?

Reply:
@timjarsky Hmm, yes that should be done. It is a bit hacky like that. I'll do that tomorrow together with the tests.

Reply:
@timjarsky Done.